pyuvdata/uvdata.py

```diff
@@ -2508,12 +2508,19 @@ def get_feedpols(self):
         else:
             return list(set(''.join(self.get_pols())))
 
-    def antpair2ind(self, ant1, ant2):
+    def _antpair2ind(self, ant1, ant2):
         """
-        Get blt indices for given (ordered) antenna pair.
+        Get blt indices for given (ordered) antenna pair. See self.get_blt_inds for
+        indices given an unordered pair.
         """
         return np.where((self.ant_1_array == ant1) & (self.ant_2_array == ant2))[0]
 
+    def antpair2ind(self, *args):
+        """ Backwards Compatability method, but will soon be deprecated """
+        warnings.warn("UVData.antpair2ind will soon be moved to UVData._antpair2ind.\n"
+                      "Try using UVData.get_blt_inds() as a replacement function.")
```

Comment:
Can you make the type of this warning a `PendingDeprecationWarning`? I think that is a little clearer